src/Model/Item.php

```diff
@@ -623,15 +637,20 @@ public static function isTooOld(array $item)
 	/**
 	 * Return the id of the given item array if it has been stored before
 	 *
-	 * @param array $item
-	 * @return integer item id
+	 * @param array $item Item record
+	 * @return integer Item id or zero on error
 	 */
-	private static function getDuplicateID(array $item)
+	private static function getDuplicateID(array $item): int
 	{
 		if (empty($item['network']) || in_array($item['network'], Protocol::FEDERATED)) {
-			$condition = ["`uri-id` = ? AND `uid` = ? AND `network` IN (?, ?, ?, ?)",
-				$item['uri-id'], $item['uid'],
-				Protocol::ACTIVITYPUB, Protocol::DIASPORA, Protocol::DFRN, Protocol::OSTATUS];
+			$condition = ['`uri-id` = ? AND `uid` = ? AND `network` IN (?, ?, ?, ?)',
```

Comment:
```suggestion
			$condition = ["`uri-id` = ? AND `uid` = ? AND `network` IN (?, ?, ?, ?)",
```

Reply:
You have `$condition = ['`` on more places:
- PostUpdate.php, lines 140, 347, 1441
- OpenWebAuthToken.php, line 87
- Item.php, line 646
- Contact.php, lines 274, 995
- DRN.php, lines 230, 2324

Do you want me to fix them all?

Reply:
No thank you, I'd rather you only focus on the files you've already edited.